app/views/dashboard.html

```diff
@@ -195,7 +196,7 @@ <h4 translate="legend.ptratio">Pupil-Teacher Ratio</h4>
               </div>
             </div>
           </div>
-          <div class="container-fluid perfomance-gauges" ng-hide="selectedSchool|exists">
+          <div class="container-fluid perfomance-gauges" ng-hide="(selectedSchoolCode|exists) || (selectedPolyId|exists)">
```

Comment:
That's a good idea... how about: if we have any new bug come in related to sidebar stuff visibility, we will move logic for that into a scope function. For now lets leave it.